resip/stack/HeaderHash.hxx

```diff
@@ -1,5 +1,6 @@
 #if !defined(RESIP_HEADERHASH_HXX)
 #define RESIP_HEADERHASH_HXX
+#include "config.h"
```

Comment:
You'll have to wrap it in
```c++
#if defined(HAVE_CONFIG_H)
#include "config.h"
#endif
```

```c++
#if defined(HAVE_CONFIG_H)
#include "config.h"
#else
#define GPERF_SIZE_TYPE unsigned int
#endif
```

Reply:
Those header files get installed by autotools right now but `config.h` does not. Without a config file header file will differ from implementation. As far as I can see those `XyzHash.hxx` files are not referenced by any other files so we maybe could exclude them from install. @sgodin what's your take on this?

Reply:
HeaderHash.hxx is referenced by Headers.cxx
MethodHash.hxx is referenced by MethodTypes.cxx
ParameterHash.hxx is referenced by ParameterTypes.cxx

Reply:
I think it would be better to install config.h as well, since:
szo@x:/usr/local/include$ rgrep config.h  |wc -l
37